--- Revision 1945
+++ Revision 1995
@@ -8,6 +8,10 @@
   border: 0;
 }
 
+.sep {
+  color: #AAA;
+}
+
 /* Short statistic */
 
 .short-stat dl {
@@ -59,12 +63,18 @@
     color: #999;
   }
   
-  table.account-overview td .status {
+  table.account-overview td.label.top {
+    vertical-align: top;
+  }
+  
+  table.account-overview td .status,
+  table.repos-table td .status {
     padding-left: 1.75em;
   	background: url(../images/i_error_light.gif) 0 0 no-repeat;
   }
   
-  table.account-overview td .status.active {
+  table.account-overview td .status.active,
+  table.repos-table td .status.success {
     padding-left: 1.75em;
   	background: url(../images/i-yes.gif) 0 0 no-repeat;
   }
